loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -237,13 +237,13 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		} else if (object) {
 			data = typeof data === 'string' ? data.replace('"', '\\"') : data;
 			var windowId = window.mobileDialogId !== undefined ? window.mobileDialogId :
-								(window.notebookbarId !== undefined ? window.notebookbarId :
-								(window.sidebarId !== undefined ? window.sidebarId : -1));
+				(window.notebookbarId !== undefined ? window.notebookbarId :
+					(window.sidebarId !== undefined ? window.sidebarId : -1));
 			var message = 'dialogevent ' + windowId
-					+ ' {\"id\":\"' + object.id
```

Comment:
@eszkadev Escaping docuble quotes inside single quotes seems redundant. eslint agrees. Wha tdo you think? Any particular reason to keep them?

Reply:
seems to be ok

Reply:
@eszkadev Thanks!